app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -104,10 +109,32 @@ fun setProfileImage(imageView: ImageView, profileAvatar: ProfileAvatar?) {
   if (profileAvatar.avatarTypeCase == ProfileAvatar.AvatarTypeCase.AVATAR_COLOR_RGB) {
     Glide.with(imageView.context)
       .load(R.drawable.ic_default_avatar)
+      .listener(object : RequestListener<Drawable> {
+        override fun onLoadFailed(
+          e: GlideException?,
+          model: Any?,
+          target: Target<Drawable>?,
+          isFirstResource: Boolean
+        ): Boolean {
+          TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
I am actually not able to understand what is this for?

Reply:
Removed comment.